src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -284,6 +289,8 @@ public static bool IsSupportedDataClassification()
 
         public static bool IsDNSCachingSetup() => !string.IsNullOrEmpty(DNSCachingConnString);
 
+        public static bool IsEnclaveAzureDatabaseSetup() => !string.IsNullOrEmpty(EnclaveAzureDatabaseConnString);
```

Comment:
The "EnclaveEnabled" keyword in config.json would mislead this case, it would be better to also include that in the condition.

Reply:
Regarding this `EnclaveEnabled` keyword, we don't actually need it for this test because the Azure DB in the connection string is always enclave-enabled. The connection string check should be enough. I will remove it in the test.

Reply:
Well, after testing locally, I have seen failures running this test since our current ColumnMasterKey generation will check the DataTestUtility.EnclaveEnabled to determine the key type (allow rich computation or not). I will add this to the test condition as you suggested.